ui/css/perf.css

```diff
@@ -449,32 +449,28 @@ li.pagination-active.active > button {
 }
 
 /* graph colors */
-.blue {
-  border-left-color: #1752b8;
+.orange {
+  border-left-color: #ffb851;
 }
 
-.green {
-  border-left-color: #19a572;
+.fire-red {
+  border-left-color: #c92d2f;
 }
 
-.darkorchid {
-  border-left-color: #9932cc;
+.cerulean {
+  border-left-color: #16bcde;
 }
 
-.scarlet {
-  border-left-color: #b81752;
+.blue-bell {
+  border-left-color: #464876;
 }
 
-.turquoise {
-  border-left-color: #17a2b8;
+.gras {
```

Comment:
Is this supposed to be "grass"? As in "grass-green"?

Reply:
![image](https://user-images.githubusercontent.com/3901809/73539595-8337f400-440c-11ea-9ef2-35305fb782db.png)


I used the coolors.co name. I thought it was a good idea to abbreviate this one for some reason.
How about `purple`?

Reply:
'gras' on its own doesn't really make sense so `purple` is fine :)

Reply:
Ok! Done!